etc/taskcluster/macos/Brewfile-build.lock.json

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
Let's also find a way to remove this file addition from the changes.

Reply:
Got it!